microbench/pom.xml

```diff
@@ -91,9 +91,20 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>test-jar</goal>
```

Comment:
Instead of building a test jar, I recommend moving all classes to `src/main` like we did for the testsuite module. You'll have to configure the surefire plugin. See https://github.com/netty/netty/blob/master/testsuite/pom.xml#L127